src/internal/observable/pairs.ts

```diff
@@ -8,7 +8,7 @@ import { Subscription } from '../Subscription';
  *
  * <span class="informal">Turn entries of an object into a stream.</span>
  *
- * <img src="./img/pairs.png" width="100%">
+ * <img src="pairs.png" width="100%">
```

Comment:
I'm really sorry, I didn't expressed well.
It should be like this:
![](pairs.png)

Reply:
makes sense. done.

Reply:
I came across another img-tag in the `generate` operator